synapse/res/username_picker/style.css

```diff
@@ -0,0 +1,179 @@
+body {
+  background: #ededf0;
+  color: #737373;
+  font-family: "Open Sans", sans-serif;
+  letter-spacing: 0.03em;
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-rows: auto 1fr; }
```

Comment:
The formatting of this file makes it very hard to find where sections end, what does it look like, by the way? I feel like it would ideally by default look like the login fallback page?

Reply:
I couldn't agree more, but one thing that was way out of scope was rewriting this.

It looks like this:

![image](https://user-images.githubusercontent.com/1389908/102523322-4958f600-408f-11eb-9446-5eb9a2e7fde6.png)

Reply:
I've stripped out all the auto-generated CSS. It now looks like this:

![image](https://user-images.githubusercontent.com/1389908/102528046-e9198280-4095-11eb-82b2-51516a5e99ce.png)

If someone else wants to make it pretty, they are welcome.